packages/product-json-to-xlsx/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@commercetools/product-json-to-xlsx",
+  "version": "0.0.1",
+  "description": "Convert commercetools products from JSON to XLSX",
+  "keywords": [
+    "commercetools",
+    "products",
+    "projections",
+    "parser",
+    "export",
+    "csv",
+    "xlsx"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/commercetools/nodejs/tree/master/packages/product-json-to-xlsx"
+  },
+  "homepage": "https://commercetools.github.io/nodejs/cli/product-json-to-xlsx.html",
+  "bugs": "https://github.com/commercetools/nodejs/issues",
+  "license": "MIT",
+  "author": {
+    "name": "Jan Juna",
+    "email": "jan.juna-ext@commercetools.de"
+  },
+  "main": "lib/index.js",
+  "bin": {
+    "product-json-to-xlsx": "bin/product-json-to-xlsx.js"
+  },
+  "files": [
+    "bin",
+    "lib"
+  ],
+  "scripts": {
+    "build": "babel src --out-dir lib"
```

Comment:
@junajan, I see linting is failing because of this line. Should be updated to the following after babel@7 migration.

`"babel src --out-dir lib --config-file '../../babel.config.js'"`

Reply:
updated here 61f002c84c51e760920fc93788af270298af520e ..